### PR TITLE
Feature Request: Talawa Library List

### LINKED ISSUE #405: Feature Request: Talawa Library List
**Is your feature request related to a problem? Please describe.**
I feel like there's no need to have a list of talawa libraries on the sidebar and in the main content again. its redundant

**Describe the solution you'd like**
I would like to remove the talawa libraries list from the main and keep the library list on the sidebar

Screenshot (28)

**Describe alternatives you've considered**

**Approach to be followed (optional)**
A clear and concise description of approach to be followed.

**Additional context**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Yass32**
**Issue Number:**
Fixes #405 

**What kind of change does this PR introduce?**
1. Removed the redundancy of 'talawa package' and library names 
Screenshot (29)

2. Fixed the padding on the sidebar so the talawa libraries name doesn't cut off 
Screenshot (31)

3. Fixed some documentation typos

**Did you add tests for your changes?**
No

**Does this PR introduce a breaking change?**
No it does not

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by Cioppolo14** [comment]
@Yass32 Please fix the first comment so that the issue automatically closes. The PR_GUIDELINES.md file has details.

Please also fill in the template for the pull request as completely as you can. I greatly helps others searching for code changes in future and helps others understand the summarized rationale for the work you have done.

**Comment by Yass32** [comment]
@Cioppolo14 is that better?

**Comment by Cioppolo14** [comment]
@Yass32 Thank you!

**Comment by Cioppolo14** [comment]
@Yass32 When you say documentation typos do you mean in the code or in the wording of the main pages?

File: static/talawa/index.html [hunk 1]
```diff
@@ -38,957 +38,57 @@
...
+      <form class="search-sidebar" role="search">
+        <input type="text" id="search-sidebar" autocomplete="off" disabled class="form-control typeahead" placeholder="Loading search...">
+      </form>
+    </header>
 
+    <ol class="breadcrumbs gt-separated dark hidden-l" id="sidebar-nav">
+      <li><a href="https://github.com/PalisadoesFoundation/talawa">talawa package</a></li>
+    </ol>
 
-    <h5 class="hidden-xs"><span class="package-name">talawa</span> <span class="package-kind">package</span></h5>
```
Comment (Cioppolo14) [inline static/talawa/index.html:991]: @Yass32 Could you tell me why this was deleted?

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
@Yass32 I left one comment and also could you point me to the documentation changes you made? Thank you!

**Comment by Yass32** [comment]
> @Yass32 When you say documentation typos do you mean in the code or in the wording of the main pages?

No sorry, I meant the wording of the main pages. But they were just a few, not too many

File: static/talawa/index.html (hunk 1 above)
Comment (Yass32) [reply static/talawa/index.html:991]: > @Yass32 Could you tell me why this was deleted?

Screenshot (34)

I felt like there was no need to repeat 'talawa package' twice


And also I created an issue to fix the 'N|solid' image but didn't get a reply

**Comment by palisadoes** [comment]
@Yass32

1. Please research this more. This file will be overwritten on the next PR in the talawa-mobile app repository.
1. Though it is in the talawa-docs repository, it is ephemeral.
1. I'll close this PR and issue until you can figure out a way to make this work with our automation

**Pull request closed without merging**